test/passing/tests/protected_object_types.ml

```diff
@@ -25,17 +25,15 @@ module Space_around = struct
   (* Ensure that the protection mechanism does not add extra spaces when
      [--space-around-*] options are sufficient. *)
 
-  module Records = struct
+  module [@ocamlformat "space-around-records = true"] Records = struct
```

Comment:
The new formatting doesn't look great for long attributes or attributes not interpreted by the compiler like this.

Reply:
I agree for attributes interpreted by the compiler, it makes sense. I am going to do a preserving version :)